src/main/java/frc/robot/utilities/TunableNumber.java

```diff
@@ -25,21 +36,22 @@ public TunableNumber(
     this.value = defaultValue;
     this.consumer = consumer;
     if (!Robot.isCompetition) {
+      inst = NetworkTableInstance.getDefault();
       numberEntry = tuningTab.add(name, defaultValue).getEntry();
+      numberEntry.close();
+      
+      doubleTopic = inst.getDoubleTopic(name);
+      doubleSub = doubleTopic.subscribe(defaultValue, PubSubOption.pollStorage(0), PubSubOption.periodic(1));
     }
   }
 
   public void periodic() {
     if (!Robot.isCompetition) {
-      consumer.accept(numberEntry.getDouble(value));
-      value = numberEntry.getDouble(value);
+      consumer.accept(doubleSub.get(defaultValue));
+      value = doubleSub.get(defaultValue);
```

Comment:
```suggestion
      value = doubleSub.get(defaultValue);
      consumer.accept(value);
```
Even though we've made it more network efficient, its still better to only request it when we need it